hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapPutTransientWithMaxIdleMessageTask.java

```diff
@@ -56,11 +58,12 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
-        return "putTransientWithMaxIdle";
+        return "put";
```

Comment:
Should this be `putTransient` because of `IMap#putTransient`?

Reply:
Yes, you are right.